src/view/pages/Login/index.tsx

```diff
@@ -69,6 +73,7 @@ class Login extends React.Component<Props, OwnState> {
   private loginPress = () => {
     // TODO: Don't just let into app
     this.props.navigateToHome();
+    this.setState({ password: '' });
```

Comment:
We already discussed the password and I will deal with this in auth. However regarding the use of `this.setState` I will copy a comment I left on Moath's PR

"Please use the function argument for `setState` ie: `this.setState(state => ({ mode: newMode }))`

I realize it makes no difference in this situation, but this is a pretty tricky bug to track down when you actually are accessing state, and the people coming after us will be reflecting our style and will not likely know the gotcha's of the framework"

Reply:
I've changed the state arguments. I've been trying to figure out how to grab the password input without writing to a variable and it might be more tricky than we expected.